Index: jinja2-2.3/jinja2/__init__.py
===================================================================
--- jinja2-2.3.orig/jinja2/__init__.py
+++ jinja2-2.3/jinja2/__init__.py
@@ -27,11 +27,11 @@
     :license: BSD, see LICENSE for more details.
 """
 __docformat__ = 'restructuredtext en'
-try:
-    __version__ = __import__('pkg_resources') \
-        .get_distribution('Jinja2').version
-except:
-    __version__ = 'unknown'
+#try:
+__version__ = 'REPLACE_ME_WITH_PROPER_VERSION'
+#        .get_distribution('Jinja2').version
+#except:
+#    __version__ = 'unknown'
 
 # high level interface
 from jinja2.environment import Environment, Template
